cardano-constitution/src/Cardano/Constitution/Config/Default.hs

```diff
@@ -0,0 +1,20 @@
+{-# LANGUAGE TemplateHaskell #-}
+module Cardano.Constitution.Config.Default
+    ( defaultConstitutionConfig
+    , ConstitutionConfig (..)
+    ) where
+
+import Cardano.Constitution.Config.Types
+import Cardano.Constitution.DataFilePaths as DFP
+
+import Data.Aeson.THReader as Aeson
+
+-- | The default params rails read from "defaultConstitutionConfig.json"
+--
+-- NOTE: this pragma should in principle be *NOINLINE*. Because of Tx we have
+-- to use *INLINABLE* instead, but I don't think it can lead to problems
+-- (besides efficiency problems, too many file reads?)
```

Comment:
I don't think it can read the file multiple times? It's a CAF, the result will get cached. It may inline the config at the call site, but that's unlikely I guess, so I think this is fine.